swiftgen-cli/ArgumentsUtils.swift

```diff
@@ -47,12 +47,20 @@ enum OutputDestination: ArgumentConvertible {
     }
   }
 
-  func write(content: String) {
+  func writeIfChanged(content: String) {
+    write(content, onlyIfChanged: true)
+  }
+  
+  func write(content: String, onlyIfChanged: Bool = false) {
     switch self {
     case .Console:
       print(content)
     case .File(let path):
       do {
+        if try path.isContentEqualTo(content) && onlyIfChanged {
```

Comment:
Btw did you check what happens if the output file doesn't exist? I mean, I hope it doesn't throw in that case — not being able to read a non-existing file — and make the whole command fail the first time you launch it :wink:

Maybe worth adding a `path.exists` test in that condition or sthg?